<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />

    <title></title>
    <link rel="stylesheet" href="dist/reveal.css" />
    <link rel="stylesheet" href="dist/theme/mattropolis.css" id="theme" />
    <link rel="stylesheet" href="plugin/highlight/zenburn.css" />
	<link rel="stylesheet" href="css/layout.css" />
	<link rel="stylesheet" href="plugin/customcontrols/style.css">
	<link rel="stylesheet" href="plugin/chalkboard/style.css">



    <script defer src="dist/fontawesome/all.min.js"></script>

	<script type="text/javascript">
		var forgetPop = true;
		function onPopState(event) {
			if(forgetPop){
				forgetPop = false;
			} else {
				parent.postMessage(event.target.location.href, "app://obsidian.md");
			}
        }
		window.onpopstate = onPopState;
		window.onmessage = event => {
			if(event.data == "reload"){
				window.document.location.reload();
			}
			forgetPop = true;
		}

		function fitElements(){
			const itemsToFit = document.getElementsByClassName('fitText');
			for (const item in itemsToFit) {
				if (Object.hasOwnProperty.call(itemsToFit, item)) {
					var element = itemsToFit[item];
					fitElement(element,1, 1000);
					element.classList.remove('fitText');
				}
			}
		}

		function fitElement(element, start, end){

			let size = (end + start) / 2;
			element.style.fontSize = `${size}px`;

			if(Math.abs(start - end) < 1){
				while(element.scrollHeight > element.offsetHeight){
					size--;
					element.style.fontSize = `${size}px`;
				}
				return;
			}

			if(element.scrollHeight > element.offsetHeight){
				fitElement(element, start, size);
			} else {
				fitElement(element, size, end);
			}		
		}


		document.onreadystatechange = () => {
			fitElements();
			if (document.readyState === 'complete') {
				if (window.location.href.indexOf("?export") != -1){
					parent.postMessage(event.target.location.href, "app://obsidian.md");
				}
				if (window.location.href.indexOf("print-pdf") != -1){
					let stateCheck = setInterval(() => {
						clearInterval(stateCheck);
						window.print();
					}, 250);
				}
			}
	};


        </script>
  </head>
  <body>
    <div class="reveal">
      <div class="slides"><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

# Introduction
## What is population genetics? what does it study?
</div></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-auto-animate="true" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

## What is biological evolution?
</div></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-auto-animate="true" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

## What is biological evolution?

Biological evolution is the change over time in the **genetic composition of a population**, which changes due to the birth and death of individuals or the migration of individuals in or out of the population.
</div></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-auto-animate="true" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

## What are the forces driving biological evolution?
</div></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-auto-animate="true" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

## What are the population **events** driving biological evolution?
- birth <!-- .element: class="fragment" -->
- death <!-- .element: class="fragment" -->
- migration <!-- .element: class="fragment" -->
</div></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

# Measuring genetic variation
</div></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-auto-animate="true" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

## Types of genetic variation
- SNP
- &shy;<!-- .element: class="fragment" data-fragment-index="1" -->microsatellites
	- &shy;<!-- .element: class="fragment" data-fragment-index="2" -->What are microsatellites?
	- &shy;<!-- .element: class="fragment" data-fragment-index="3" -->what are they useful for?
- &shy;<!-- .element: class="fragment" data-fragment-index="4" -->CNV
</div></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

## Types of genetic variation

<div class="callout callout-color-default">
<div class="callout-title">
<div class="callout-icon">

<i class="fas fa-pencil-alt" ></i>


</div>
<div class="callout-title-inner">

Haplotype

</div>
</div>
<div class="callout-content">

refer to the combination of alleles at multiple loci on the same chromosomal homolog.

</div>
</div>
</div></script></section><section ><section data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

### How do you draw the network (b) based on the alignment (a)?

<img src="10-asset/11-attachment/20240502-IntroGenetAnal-18-4.png" alt="" style="width: 800px; object-fit: fill">
</div></script></section><section data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<img src="10-asset/11-attachment/20240502-IntroGenetAnal-18-5a.png" alt="" style="object-fit: scale-down">
</div></script></section><section data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

### Distribution of the star-cluster haplotype
<img src="10-asset/11-attachment/20240502-IntroGenetAnal-18-5b.png" alt="" style="object-fit: scale-down">
</div></script></section></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

## How do you quantify the level of genetic variation?
<img src="10-asset/11-attachment/20240502-IntroGenetAnal-18-4.png" alt="" style="width: 800px; object-fit: fill">


<div class="has-dark-background" style="background-color: forestgreen; position: absolute; left: 7.5%; top: 55%; height: 45%; width: 85%; display: flex; flex-direction: column; align-items: center; justify-content: center" >


- count the number of variable sites
- &shy;<!-- .element: class="fragment" data-fragment-index="1" -->account for the length of the region (sequence)
- &shy;<!-- .element: class="fragment" data-fragment-index="2" -->account for the number of sequences (sample size)
- &shy;<!-- .element: class="fragment" data-fragment-index="3" -->Average # of differences between two sequences
</div>
</div></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

## Level of genetic variation across domains of life
<img src="10-asset/11-attachment/20240505-Lefler-2012-fig1.png" alt="" style="object-fit: scale-down">


<div class="has-light-background" style="background-color: white; position: absolute; left: 50%; top: 20%; height: 80%; width: 50%; display: flex; flex-direction: column; align-items: center; justify-content: center" >
<!-- .element: class="fragment" -->
<img src="10-asset/11-attachment/20240204-Intro to Genetic Analysis 11th ed-fig-18-15.png" alt="" style="object-fit: scale-down">

<p style="color:black">M. Lynch and J. S. Conery, Science 2003</p>
</div>

<div class="footnotes" role="doc-endnotes">
<ol>
<li id="fn:1" role="doc-endnote" class="footnote"><p>

Leffler et al. 2012. “Revisiting an Old Riddle: What Determines Genetic Diversity Levels within Species?” _PLoS Biology_ 10 (9): e1001388. [https://doi.org/10.1371/journal.pbio.1001388](https://doi.org/10.1371/journal.pbio.1001388).

</p></li></ol>
</div>
</div></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-auto-animate="true" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

## Gene pool and genotype frequencies
<split left="2" right="1">
<img src="10-asset/11-attachment/20240504-IntroGenetAnal-18-7.png" alt="" style="width: 600px; object-fit: fill">


- what is a gene pool?
- &shy;<!-- .element: class="fragment" data-fragment-index="1" --><mark>**Locus**</mark>: "region of interest", may be an entire gene, or a single nucleotide
- &shy;<!-- .element: class="fragment" data-fragment-index="2" --><mark>**Allele**</mark>: different "versions" of the sequence existing in a population
- &shy;<!-- .element: class="fragment" data-fragment-index="3" --><mark>**Genotype**</mark>: denotes the total allele combinations at a locus, e.g., two alleles for diploids. <br><br>
- &shy;<!-- .element: class="fragment" data-fragment-index="4" -->What are the genotype frequencies?
- &shy;<!-- .element: class="fragment" data-fragment-index="5" -->What are the allele frequencies?
</split>
</div></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-auto-animate="true" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

## Gene pool and genotype frequencies
<split left="2" right="1">
<img src="10-asset/11-attachment/20240504-IntroGenetAnal-18-7.png" alt="" style="width: 600px; object-fit: fill">


- Knowing the allele frequency (`$p_A = 18/32$`), what is a "reasonable guess" of the genotype frequency (e.g., for `$f_{A/A}$`)?
- &shy;<!-- .element: class="fragment" data-fragment-index="1" -->How close is it to the observed genotype frequency?
- &shy;<!-- .element: class="fragment" data-fragment-index="2" -->Now reflect on your intuitive guess, what assumptions need to be made for it to work?
</split>
</div></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-auto-animate="true" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

## Hardy-Weinberg Equilibrium
    Suppose allele frequency of A = p, and frequency of a = q

| `$f_{A/A}$` | `$f_{A/a}$` | `$f_{a/a}$` |
| --------- | --------- | --------- |
| `$p^2$`     | `$2pq$`     | `$q^2$`     |
- &shy;<!-- .element: class="fragment" data-fragment-index="1" -->Assume the allele frequency of A is the same in females and males.
- &shy;<!-- .element: class="fragment" data-fragment-index="2" -->The probability of a baby frog to have A/A genotype is `$p^2$`
- &shy;<!-- .element: class="fragment" data-fragment-index="3" -->What other assumptions are needed?
- &shy;<!-- .element: class="fragment" data-fragment-index="4" --><mark>Will any evolutionary force, such as migration and natural selection, affect the Hardy-Weinberg Equilibrium?</mark>
</div></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-auto-animate="true" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

## Hardy-Weinberg Equilibrium

<split left="1" right="2">

<div class="mermaid">
graph TD
	A[sexually mature]
	B[gene pool]
	C[new zygote]
	A-->|generate gamete|B
	B-->|mating, HWE|C
	C-->|selection etc.|A

</div>



- &shy;<!-- .element: class="fragment" data-fragment-index="1" -->HWE only refers to the step of forming new zygotes.
- &shy;<!-- .element: class="fragment" data-fragment-index="2" -->Only assumptions needed are 
	- &shy;<!-- .element: class="fragment" data-fragment-index="3" -->random mating **with respect to the locus of interest**
	- &shy;<!-- .element: class="fragment" data-fragment-index="4" -->same allele frequencies in both sexes
- &shy;<!-- .element: class="fragment" data-fragment-index="5" -->Under these assumptions, HWE is restored _instantaneously_ upon mating
- &shy;<!-- .element: class="fragment" data-fragment-index="6" -->Often hear about "no selection, no migration" only needed for HWE frequencies to represent a **long term equilibrium**.
</split>
</div></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

### HWE exercise
<split left="3" right="1">
<div class="callout callout-color4">
<div class="callout-title">
<div class="callout-icon">

<i class="fas fa-question-circle" ></i>


</div>
<div class="callout-title-inner">

Question

</div>
</div>
<div class="callout-content">

On the coastal islands of British Columbia there is a subspecies of black bear (*Ursus americanus kermodei*, Kermode’s bear). Many members of this black bear subspecies are white; they’re sometimes called spirit bears. These bears aren’t hybrids with polar bears, nor are they albinos. They are homozygotes for a recessive change at the MC1R gene. Individuals who are GG at this SNP are white, while AA and AG individuals are black.

The genotype counts for the MC1R polymorphism in a sample of bears from British Columbia’s island populations from Ritland et al. (2001) are: `$AA=42, AG=24, GG=21$`

What are the expected frequencies of the three genotypes under HWE?

</div>
</div>

<img src="10-asset/11-attachment/20240506-Coop-fig2.5.png" alt="" style="width: 100px; object-fit: fill">


</split>
</div></script></section></div>
    </div>

    <script src="dist/reveal.js"></script>

    <script src="plugin/markdown/markdown.js"></script>
    <script src="plugin/highlight/highlight.js"></script>
    <script src="plugin/zoom/zoom.js"></script>
    <script src="plugin/notes/notes.js"></script>
    <script src="plugin/math/math.js"></script>
	<script src="plugin/mermaid/mermaid.js"></script>
	<script src="plugin/chart/chart.min.js"></script>
	<script src="plugin/chart/plugin.js"></script>
	<script src="plugin/menu/menu.js"></script>
	<script src="plugin/customcontrols/plugin.js"></script>
	<script src="plugin/chalkboard/plugin.js"></script>

    <script>
      function extend() {
        var target = {};
        for (var i = 0; i < arguments.length; i++) {
          var source = arguments[i];
          for (var key in source) {
            if (source.hasOwnProperty(key)) {
              target[key] = source[key];
            }
          }
        }
        return target;
      }

	  function isLight(color) {
		let hex = color.replace('#', '');

		// convert #fff => #ffffff
		if(hex.length == 3){
			hex = `${hex[0]}${hex[0]}${hex[1]}${hex[1]}${hex[2]}${hex[2]}`;
		}

		const c_r = parseInt(hex.substr(0, 2), 16);
		const c_g = parseInt(hex.substr(2, 2), 16);
		const c_b = parseInt(hex.substr(4, 2), 16);
		const brightness = ((c_r * 299) + (c_g * 587) + (c_b * 114)) / 1000;
		return brightness > 155;
	}

	var bgColor = getComputedStyle(document.documentElement).getPropertyValue('--r-background-color').trim();
	var isLight = isLight(bgColor);

	if(isLight){
		document.body.classList.add('has-light-background');
	} else {
		document.body.classList.add('has-dark-background');
	}

      // default options to init reveal.js
      var defaultOptions = {
        controls: true,
        progress: true,
        history: true,
        center: true,
        transition: 'default', // none/fade/slide/convex/concave/zoom
        plugins: [
          RevealMarkdown,
          RevealHighlight,
          RevealZoom,
          RevealNotes,
          RevealMath.MathJax3,
		  RevealMermaid,
		  RevealChart,
		  RevealCustomControls,
		  RevealMenu,
		  RevealChalkboard, 
        ],


    	allottedTime: 120 * 1000,

		mathjax3: {
			mathjax: 'plugin/math/mathjax/tex-mml-chtml.js',
		},
		markdown: {
		  gfm: true,
		  mangle: true,
		  pedantic: false,
		  smartLists: false,
		  smartypants: false,
		},

		mermaid: {
			theme: isLight ? 'default' : 'dark',
		},

		customcontrols: {
			controls: [
				{ icon: '<i class="fa fa-pen-square"></i>',
				title: 'Toggle chalkboard (B)',
				action: 'RevealChalkboard.toggleChalkboard();'
				},
				{ icon: '<i class="fa fa-pen"></i>',
				title: 'Toggle notes canvas (C)',
				action: 'RevealChalkboard.toggleNotesCanvas();'
				},
			]
		},
		menu: {
			loadIcons: false
		}
      };

      // options from URL query string
      var queryOptions = Reveal().getQueryHash() || {};

      var options = extend(defaultOptions, {"width":960,"height":700,"margin":0.04,"controls":true,"progress":true,"slideNumber":false,"transition":"fade","transitionSpeed":"default"}, queryOptions);
    </script>

    <script>
      Reveal.initialize(options);
    </script>
  </body>

  <!-- created with Advanced Slides -->
</html>
